weblate/formats/ttkit.py

```diff
@@ -16,7 +16,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
-"""Translate Toolkit based file format wrappers."""
+"""Translate Toolkit-based file-format wrappers."""
```

Comment:
it's "based upon Translate Toolkit", not "to Translate... Toolkit-based". Better leave it without a dash or use something like this:

```suggestion
"""TranslateToolkit-based file-format wrappers."""
```

Reply:
keeping original